apps/server/src/modules/account/domain/services/account.service.spec.ts

```diff
@@ -579,6 +579,14 @@ describe('AccountService', () => {
 		});
 	});
 
+	describe('updateFailedLogin', () => {
+		it('should call updateLastLogin in accountServiceDb', async () => {
+			await accountService.updateLastLogin('accountId', new Date());
```

Comment:
Hier könnte man noch auf new ObjectId().toHexString() umbauen. Falls irgendwann jemand auf die Idee kommt in einer factory den typ vor dem persistieren zu prüfen.